Lecture_1/Exercise_5/calc_lim.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+#include <stdint.h>
+
+int main(void){
+    uint64_t n=0,i;
+    
+    n--;
+    for(i=2;n>0;i++)
+	n/=i;
+    printf("%d\n",i);
+    return 0;
+}
```

Comment:
Looks like the file is not  from here)

Reply:
This file i use fot calculate max possible argument for factorial function. ;-)